### DISCUSSION TITLE
Toggle on valid target?

### DISCUSSION THREAD

**Discussion by DeathspectreNZ** [Hekili/hekili#1497] [General]
Is there a way to toggle the Queue to only show on valid targets?  I read it was made available in 8.3, but can not find any options to do so.

**Comment by Hekili** [comment]
You can set Visibility options for each Display independently.  If you specify "Advanced," you can use the sliders to specify its transparency by default (Always), with a target (Target), when in combat (Combat), or when in combat and having a specified target (Combat w/ Target).

Setting Always to 0 and Target to 1 will result in the display being hidden out of combat, but fully visible when you have a target.

You can also use the "Multiple" display to adjust these values on all of your displays at once (check them all at the top of the panel, then update your Visibility settings).

**Comment by DeathspectreNZ** [comment]
Awesome, thanks for the info.
